docs/source/developer/numba-runtime.rst

```diff
@@ -176,6 +176,19 @@ Here is an example that uses the ``nrt_external.h``:
       return mi;
   }
 
+It is important to ensure that the NRT is initialized prior to making calls to
+it, calling ``numba.core.runtime.nrt.rtsys.initialize(context)`` from Python
+will have the desired effect. Similarly the code snippet:
+
+.. code-block:: Python
+
+  from numba.core import typing, cpu
+  typingctx = typing.Context()
+  cpu.CPUContext(typingctx, 'cpu')
```

Comment:
Better not to suggest users to create a fresh `CPUContext`. They should get and initialize the singleton with `numba.core.registry.cpu_target.target_context`